libcxx/include/__atomic/contention_t.h

```diff
@@ -19,7 +19,7 @@
 
 _LIBCPP_BEGIN_NAMESPACE_STD
 
-#if defined(__linux__) || (defined(_AIX) && !defined(__64BIT__))
+#if defined(__linux__) || defined(__Fuchsia__) || (defined(_AIX) && !defined(__64BIT__))
```

Comment:
Might be bit unfortunate, but isn’t this an ABI break for your platform ?

Reply:
That's not really an issue on Fuchsia.